src/main/java/net/pms/dlna/DLNAMediaAudio.java

```diff
@@ -166,7 +166,7 @@ public boolean isAACLC() {
 	/**
 	 * @return True if the audio codec is AAC HE.
 	 */
-	public boolean isAACHE() {
+	public boolean isHEAAC() {
```

Comment:
Why did you rename this? We have AACLC, what's the logic in calling it HEAAC all of a sudden? Especially since the FormatConfiguration is called. AAC_HE and the string "aac-he".